example/themes/original/src/components/SingleOrderCard/index.tsx

```diff
@@ -172,10 +172,17 @@ const SingleOrderCardUI = (props: SingleOrderCardParams) => {
         <InnerContainer>
           {(!!order.business?.logo || theme?.images?.dummies?.businessLogo) && (
             <Logo style={styles.logoWrapper}>
-              <OIcon
-                url={optimizeImage(order.business?.logo || theme?.images?.dummies?.businessLogo, 'h_300,c_limit')}
-                style={styles.logo}
-              />
+              {!!order.business?.logo ? (
```

Comment:
Refactor this implementation, You're repeating the same code with a simple validation